src/web/error.rs

```diff
@@ -59,6 +61,13 @@ impl Handler for Nope {
                         .to_resp("releases")
                 }
             }
+            Nope::InternalServerError => {
+                // something went wrong, details should have been logged
+                Page::new("internal server error".to_owned())
+                    .set_status(status::InternalServerError)
+                    .title("Internal server error")
+                    .to_resp("error")
```

Comment:
Do we want to link to the GitHub page so people can make an issue?

Reply:
Might be a good followup issue to do once the template changes are in, all the error pages could do with a bit of a sprucing up.

Reply:
I don't think asking to open an issue on GitHub this way is much beneficial for us: without a backtrace it's hard to then debug those issues.

A thing we could do is generate a random UUID, print it in the same line as the error and show it in the error page, so when people open the issue we can simply do a `journalctl -u docs.rs | grep UUID` to get the exact error the user encountered.

Reply:
The UUID sounds useful but it can go in a follow-up, for now I want to get the bug fix in.